External/TestDataSet/sourceStructure.json

```diff
@@ -0,0 +1,5 @@
+{"folderHierarchy":["^(.)+$","^(ASL|T1w|M0|T2|FLAIR)$"],
```

Comment:
We don't need `sourceStructure.json` when there are no DICOM data.

Reply:
I thought we could also add a `rawdata` version, so that users can also test at least one step of the import pipeline and they have an exemplary bids dataset. But you're right, `sourceStructure.json` isn't needed. We just have not example to show to users right now and if they have no examples, it's kind of hard to explain these things. We can't refer everyone to the flavor database.

Reply:
Yes. Of course, if we add raw_data - then this is needed.